go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -933,9 +936,35 @@ func verifyClusterHealth(t *testing.T, cluster *VitessCluster) {
 	iterateTablets(t, cluster, checkTabletHealth)
 }
 
+const acceptableLagSeconds = 5
+
+func waitForLowLag(t *testing.T, keyspace, workflow string) {
```

Comment:
Possible to share this code between the two end2end tests? If not, see my earlier comments about this code in `sharded_buffer_test.go`

Reply:
Yeah, classic anti-pattern :( Because they are in different packages and we don't yet have a `utils` package to reuse such code, I took the short-cut. Merging as is now to catch the release deadline, will address "soon".